src/pages/new-vehicle.vue

```
@@ -25,72 +58,205 @@
+  try {
+    if (!vehicleData.value.owner.uuid) {
+      snackbar.Error("Please select an owner first");
+      return;
+    }
+
+    if (!vehicleData.value.registration) {
+      snackbar.Error("Please enter a registration plate number");
+      return;
+    }
+
+    const response = await createVehicle(
```

ESLint

[error] 'response' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)